### DISCUSSION TITLE
Data directory for Docker

### DISCUSSION THREAD

**Discussion by DataHearth** [requarks/wiki#4879] [Help / Questions] [answered]
Hello there! 

I'm switching from the `linuxserver/wikijs` image to the official one. 

I've followed the installation tutorial for docker but it doesn't mention where are stored uploaded files (and also configuration file `config.yml`). 

As I walked through the [Dockerfile](https://github.com/Requarks/wiki/blob/2.5.117/dev/build/Dockerfile), I've seen that wiki is mapping all its data to `/wiki`. But when I launch the container, the volume that I mapped `/mnt/user/wiki:/wiki` is totally empty. I should see the server itself + `cache` and `uploads` folder.

So I went inside and saw that the app itself is on `/app/wiki` and `cache + uploads` are here `/data`. Is that normal ? If so, I think it should be updated inside the Docker installation steps ^^. Although, `uploads` folder is empty after upload an image. I still don't know where they're storage. Can someone help me to figure out ?

Thanks!

**Comment by maeries** [comment]
I'm wondering the same. After recreating the containerl all my uploaded files are still there, even though the wiki service doesn't have any volumes. Are the files stored in the Database?

Also since there is a `VOLUME ["/wiki/data/content"]` in the dockerfile, but no volume mapped to `/wiki/data/content` in the docker-compose.yml, I get a new directory in `/var/lib/docker/volumes` every time I recreate the container

**Comment by NGPixel** [comment, marked as answer]
Assets are stored in the database as blobs.
